### DISCUSSION TITLE
Multiple Dropbox accounts

### DISCUSSION THREAD

**Discussion by mbriney** [iterate-ch/cyberduck#14795] [Q&A]
**Describe the bug**
You can no longer specify the email address of the Dropbox account when adding a new account.  As a result, it no longer allows you to have multiple Dropbox accounts.  At least not using the UI.  

**To Reproduce**
1. Have a setup that has an existing Dropbox account
2. Go to open connection
3. Select dropbox
4. The account is now a dropdown and is greyed out and defaulting to the existing first dropbox account you've added

**Expected behavior**
It should instead allow you to manually enter your dropbox email, if that account is not in the keychain it should prompt you to generate a new oAuth.  At least that's how it used to work.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS 13.4

**Comment by tatze96** [comment]
The credential fields of protocols that require an OAuth code for account authorization are purposely greyed out because you'll get redirected to an OAuth page within the web browser on the first connection of the bookmark. The credential fields will be filled in automatically once the authorization process is finished.

If the bookmark defaults to an existing bookmark's credentials, you most likely have keychain entries with an empty username. Please clear the corresponding entries with empty usernames from *Keychain Access.app* on macOS or, respectively, *Windows Credential Manager* on Windows and try again to connect to the additional account.
